app/scripts/cart/cart.directives.ts

```diff
@@ -439,7 +439,8 @@ module ngApp.cart.directives {
                 + "<a href='https://gdc.nci.nih.gov/access-data/gdc-data-transfer-tool' target='_blank'>"
                 +   "GDC Data Transfer Tool"
                 + "</a>"
-                + " to continue."
+                + " to continue.",
```

Comment:
No I mean instead of 

    warning: () => "Your cart contains more than 5GBs of data. <br />"
                + "Please select the \"Download > Manifest\" option and use the "
                + "<a href='https://gdc.nci.nih.gov/access-data/gdc-data-transfer-tool' target='_blank'>"
                +   "GDC Data Transfer Tool"
                + "</a>"
                + " to continue.",

this:

    warning: () =>
      `Your cart contains more than 5GBs of data. <br />.
        Please select the "Download > Manifest" option and use the
        <a href='https://gdc.nci.nih.gov/access-data/gdc-data-transfer-tool' target='_blank'>
          GDC Data Transfer Tool
        </a> to continue`